python/paddle/fluid/tests/unittests/test_layers.py

```diff
@@ -456,182 +456,6 @@ def test_matmul(self):
         np.testing.assert_allclose(static_ret, dy_ret_value, rtol=1e-05)
         np.testing.assert_allclose(static_ret, dy_eager_ret_value, rtol=1e-05)
 
-    def test_conv2d(self):
-        with self.static_graph():
-            images = layers.data(name='pixel', shape=[3, 5, 5], dtype='float32')
-            ret = layers.conv2d(input=images, num_filters=3, filter_size=[2, 2])
```

Comment:
shall we delete test of `layers.conv2d` this time, or with `layers.conv2d` delete ?

Reply:
This unittest is to check whether the results of  `layers.conv2d` and `dygraph.Conv2D` are same.  There is no need to keep this unittest here since the latter one will be removed.  


And for api `layers.conv2d`, there are `test_functional_conv2d.py`, `test_conv2d_layer.py` and `test_conv2d_op.py` to check its correctness.